product-bundles-bulk-discounts-for-woocommerce.php

```diff
@@ -445,6 +451,51 @@ public static function save_meta( $product ) {
 		}
 	}
 
+	/**
+	 * Export bulk discounts as formatted meta data.
+	 *
+	 * @param  string        $meta_value
+	 * @param  WC_Meta_Data  $meta
+	 * @return string        $meta_value
+	 */
+	public static function export_bulk_discounts( $meta_value, $meta ) {
+
+		if ( '_wc_pb_quantity_discount_data' === $meta->key ){
+			$meta_value = json_encode( maybe_unserialize( $meta_value ) );
+
+		}
+
+		return $meta_value;
+	}
+
+	/**
+	 * Parse and import bulk discounts.
+	 *
+	 * @param  array  $parsed_data
+	 * @return array  $parsed_data
+	 */
+	public static function import_bulk_discounts( $parsed_data ) {
+
+		if ( empty( $parsed_data[ 'meta_data' ] ) ) {
+			return $parsed_data;
+		}
+
+		foreach ( $parsed_data[ 'meta_data' ] as $index => $meta_data ) {
+
+			if ( '_wc_pb_quantity_discount_data' === $meta_data[ 'key' ] ) {
```

Comment:
If the `_wc_pb_quantity_discount_data` is only appearing once in the `meta_data` array, i would opt for a `break` statement at the end of this `if` to avoid looping through all the remaining meta data keys.